Bottom: c4de037755ff3b3e6443810669741dcb4f15b609
Top:    b91a15df11d9bdd99579ac310aaee0333a21af9f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-07 22:15:58 -0400

Refresh of dma-module

---

diff --git a/sound/soc/fsl/mpc5200_dma.c b/sound/soc/fsl/mpc5200_dma.c
index 6d4b8b8..763be11 100644
--- a/sound/soc/fsl/mpc5200_dma.c
+++ b/sound/soc/fsl/mpc5200_dma.c
@@ -1,5 +1,5 @@
 /*
- * Freescale MPC5200 Audio DMA
+ * Freescale MPC5200 PSC DMA
  * ALSA SoC Platform driver
  *
  * Copyright (C) 2008 Secret Lab Technologies Ltd.
